### DISCUSSION TITLE
Using Stanza to negate words

### DISCUSSION THREAD

**Discussion by Felixda1** [microsoft/presidio#833] [Q&A]
Hi,
I'm wondering if anyone would be able to point me to a good tutorial/article/answer my thoughts in general about using Stanza to make Presidio ignore certain words in the text. I'm using stanza to pick up certain medical terms and when it does, I want Presidio to **not** redact that information. EX. If stanza picks up a word with xpos as a proper noun, I want Presidio to ignore it during the PERSON filtering. (I know that's not medical and may be counterintuitive, it was just an example as I'm still working on the specific tags.)

**Comment by omri374** [comment]
Hi @Felixda1, there is no built-in support in Presidio for this. Can post-processing on the responses of Presidio Analyzer help? If you have the same span identified as a medical term and as a PII, remove the PII.
